### ISSUE TITLE
Not enough votes from Holy Land belief

### ISSUE THREAD

**Issue by BiteInTheMark** [LoneGazebo/Community-Patch-DLL#4856]
_1. Mod version (i.e Date - 4/23):_
10-10

_2. Mod list (if using Vox Populi only, leave blank):_
blank

_3. Error description:_ 
Not more than 4 votes by the reformation belief Holy Land, even I have 9 Holy Sites an 4 Landmarks.
Edit: after I have placed the 10th holy site, I got 5 votes. So it looks like Landmarks didnt give any votes,

_4. Steps to reproduce (optional):_
Creating a lot of holy sites in your empire, while you have the reformation belief Holy Land.

---------------------------
Supporting information:
Please note that you can attach .zip files by dragging-and-dropping them. If possible, zip up all supporting data and post that way.

1. Log files  [Lua.log](https://github.com/LoneGazebo/Community-Patch-DLL/files/2522879/Lua.log)
[Database.log](https://github.com/LoneGazebo/Community-Patch-DLL/files/2522888/Database.log)
[Gamecore.log](https://github.com/LoneGazebo/Community-Patch-DLL/files/2522889/Gamecore.log)
[xml.log](https://github.com/LoneGazebo/Community-Patch-DLL/files/2522890/xml.log)

2. Save game (always attach a save that was made a turn before the error; located at My Documents/My Games/Sid Meier's Civilization 5/ModdedSaves):
https://workupload.com/file/kGdMr8XX
Ive played the game with a modpack. Only pure VP mods

4. Screenshots (optional):
https://www.bilder-upload.eu/upload/203b5e-1540740715.jpg
https://www.bilder-upload.eu/upload/7ef334-1540740745.jpg

**Comment by cdanger341** [comment]
I've run into this bug too. It seems that Holy Land is using the higher of either Holy Site or Landmark counts. In my last game I had 4 Holy Sites by the time I unlocked Archaeology and had 6 Historic Sites within workable range of my cities. My # of delegates stayed at +2 from the effects of Holy Land until I had turned all 6 sites into Landmarks, at which point it turned into +3. Creating my 5th and 6th Holy Site did not increase the # of delegates, and the game ended before I could create 8 Holy Sites to see if that would bump the delegate count to +4.
